src/utils/getUnitsAmount.ts

```diff
@@ -0,0 +1,10 @@
+export function getUnintsAmount(unintsAmount: number) {
+  if (unintsAmount > 20) {
+    return "restam poucas unidades";
+  }
+  if (unintsAmount === 1) {
+    return "somente mais 1 unidade";
+  } else {
+    return "faltam " + unintsAmount + " unidades";
+  }
```

Comment:
```suggestion
  return (unitsAmount ===1) ? 'resta apenas 1 unidade' : `restam ${unitsAmount} unidades`;
```
Let's try to don't forget best practices and code improvement

Reply:
Also:
- unintsAmount or unitsAmount?
- units = unidade // amount = quantidade (quantidade de unidades?)
- let's change to: couponsRemaining and getCouponsRemaining

Reply:
done